_authentication.md

```diff
@@ -244,4 +244,6 @@ curl https://api.uphold.com/v0/me \
 
 You can use Basic Authentication by providing your username or email and password combination.
 
-If OTP (One Time Password, also known as Two-Factor Authentication) is required, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `OTP-Token: Required`. Execute the command above again, this time passing your OTP verification code as a header, like so: `OTP-Token: <OTP-Token>`.
+If OTP (One-Time Password, also known as Two-Factor Authentication) is required, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `OTP-Token: Required` or `OTP-Method-Id: Required`.
+For the former case, execute the command above again, this time passing your OTP verification code and phone id as a headers, like so: `OTP-Phone-Id: <OTP-Phone-Id>` (deprecated) and `OTP-Token: <OTP-Token>`.
```

Comment:
If this is deprecated, I don't think we should be documenting it here. Let's encourage developers to use the non-deprecated API.

Reply:
I disagree, if we support both cases then we have to have both cases documented. I think we should only drop this, when/if we drop support for Authy. New developers coming in to our documentation should know how to be able to deal with authenticating users with accounts still using Authy.

Reply:
We still support authy with `OTP-Method-Id`, so we should encourage developers to use that. `OTP-Phone-Id` is deprecated for developers and they should use `OTP-Method-Id` instead, which replaces it.

Reply:
I was unaware of that fact, in that case, I agree that we can drop the deprecated header from this text.

Reply:
Done.